packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.scss

```diff
@@ -0,0 +1,62 @@
+@use 'usa-step-indicator/src/styles/usa-step-indicator';
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@import '../../mixins/accessibility.css';
+
+@import '../../global/formation_overrides';
+
+
+
+.usa-step-indicator__segment::after {
+  background-color: #919191; // USWDS design and sketch
+}
+@media (min-width: 40em) {
+  .usa-step-indicator--counters .usa-step-indicator__segment::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment::before {
+    box-sizing: border-box;
+    box-shadow: inset 0 0 0 rem-override(0.25rem) #919191, 0 0 0 rem-override(0.25rem) #fff // USWDS design and sketch
+  }
+
+  .usa-step-indicator--counters .usa-step-indicator__segment--current::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment--current::before,
+  .usa-step-indicator--counters .usa-step-indicator__segment--complete::before, .usa-step-indicator--counters-sm .usa-step-indicator__segment--complete::before {
+    box-shadow: 0 0 0 rem-override(0.25rem) #fff;
```

Comment:
The only place we should be using `rem-override` is in the formation override stylesheet.  Just use `px` in the component stylesheet.